reference/orders.v2.oas2.yml

```
@@ -3677,14 +3683,15 @@ components:
           enum:
             - auspost
             - canadapost
+            - carrier_{your_carrier_id} (only used if you have set up a carrier using the Shipping Provider API)
```

```suggestion
            - carrier_{your_carrier_id} (only used if the carrier is a [third-party Shipping Provider](/api-docs/providers/shipping))
```
@bc-tgomez Not sure if you want to move the parentheses portion to the description instead